### ISSUE TITLE
Plotting fails for large trees

### ISSUE THREAD

**Issue by bubzebub** [ndphillips/FFTrees#4]
When I try to plot trees where the max.levels > 4, I get errors such as:

Error: object 'ball.box.width' not found

Error in plot.fft(fit.fft, which.tree = 8) : 
  object 'plot.width' not found

Error: object 'ball.box.width' not found
In addition: Warning messages:
1: In if (a.balls > 0) { :
  the condition has length > 1 and only the first element will be used
2: In 1:a.balls : numerical expression has 7 elements: only the first used
3: In 1:a.balls : numerical expression has 7 elements: only the first used
4: In if (b.balls > 0) { :
  the condition has length > 1 and only the first element will be used
5: In 1:b.balls : numerical expression has 7 elements: only the first used
6: In 1:b.balls : numerical expression has 7 elements: only the first used

**Comment by ndphillips** [comment]
Can you attach a dataset (either as .RData or a .txt) with your code that causes the issue?

If you don't want to share the data, can you show me the trees in your fft object with `summary(fft.object)`?

**Comment by bubzebub** [comment]
Unfortunately I can't share these data.  Here is the output:

```
> summary(fit.fft)
   tree.num                                    level.name
1         1                      LCS;FullCallTime;weekeve
5         2             LCS;FullCallTime;weekeve;weeknite
3         3                              LCS;FullCallTime
2         4             LCS;FullCallTime;weekeve;weeknite
10        5 LCS;FullCallTime;weekeve;weeknite;dialcounter
6         6 LCS;FullCallTime;weekeve;weeknite;dialcounter
14        7 LCS;FullCallTime;weekeve;weeknite;dialcounter
4         8             LCS;FullCallTime;weekeve;weeknite
8         9             LCS;FullCallTime;weekeve;weeknite
16       10 LCS;FullCallTime;weekeve;weeknite;dialcounter
                               level.class  level.exit level.threshold level.sigdirection
1                  numeric;numeric;numeric     0;0;0.5           2;0;0             >;>;<=
5          numeric;numeric;numeric;numeric   0;0;1;0.5         2;0;0;1           >;>;<=;<
3                          numeric;numeric       0;0.5             2;0                >;>
2          numeric;numeric;numeric;numeric   1;0;0;0.5         2;0;0;1           >;>;<=;<
10 numeric;numeric;numeric;numeric;numeric 1;0;0;1;0.5    2;0;0;1;3.37         >;>;<=;<;<
6  numeric;numeric;numeric;numeric;numeric 1;0;1;0;0.5    2;0;0;1;3.37         >;>;<=;<;<
14 numeric;numeric;numeric;numeric;numeric 1;0;1;1;0.5    2;0;0;1;3.37         >;>;<=;<;<
4          numeric;numeric;numeric;numeric   1;1;0;0.5         2;0;0;1           >;>;<=;<
8          numeric;numeric;numeric;numeric   1;1;1;0.5         2;0;0;1           >;>;<=;<
16 numeric;numeric;numeric;numeric;numeric 1;1;1;1;0.5    2;0;0;1;3.37         >;>;<=;<;<
   n.train hi.train mi.train fa.train cr.train  hr.train  far.train   v.train dprime.train
1   220599    12905    33889     7026   166779 0.2757832 0.04042461 0.2353586    0.5751826
5   220599    17486    29308    12040   161765 0.3736804 0.06927304 0.3044074    0.5795529
3   220599    21496    25298    24688   149117 0.4593751 0.14204424 0.3173309    0.4845860
2   220599    27455    19339    37181   136624 0.5867205 0.21392365 0.3727969    0.5059988
10  220599    31360    15434    45587   128218 0.6701714 0.26228820 0.4078832    0.5383467
6   220599    35816    10978    59839   113966 0.7653973 0.34428814 0.4211091    0.5622802
14  220599    38806     7988    74065    99740 0.8292944 0.42613849 0.4031559    0.5687972
4   220599    43579     3215   121341    52464 0.9312946 0.69814447 0.2331501    0.4832156
8   220599    45935      859   144207    29598 0.9816429 0.82970570 0.1519372    0.5679631
16  220599    46114      680   147212    26593 0.9854682 0.84699520 0.1384730    0.5794962
   n.test hi.test mi.test fa.test cr.test   hr.test   far.test    v.test dprime.test
1   94543    5439   14519    3041   71544 0.2725223 0.04077227 0.2317500   0.5682957
5   94543    7420   12538    5216   69369 0.3717807 0.06993363 0.3018471   0.5745724
3   94543    9069   10889   10691   63894 0.4544042 0.14333981 0.3110644   0.4754462
2   94543   11534    8424   16060   58525 0.5779136 0.21532480 0.3625888   0.4923197
10  94543   13189    6769   19580   55005 0.6608378 0.26251927 0.3983185   0.5251742
6   94543   15053    4905   25647   48938 0.7542339 0.34386271 0.4103712   0.5449088
14  94543   16418    3540   31775   42810 0.8226275 0.42602400 0.3966035   0.5559654
4   94543   18545    1413   51949   22636 0.9292013 0.69650734 0.2326940   0.4777436
8   94543   19578     380   61740   12845 0.9809600 0.82778038 0.1531796   0.5642813
16  94543   19669     289   63048   11537 0.9855196 0.84531742 0.1402022   0.5837322
```

**Comment by ndphillips** [comment]
@bubzebub I think I fixed the error. Please install the latest version and try again...

`devtools::install_github("ndphillips/FFTrees", build_vignette = T)`

**Comment by ndphillips** [comment]
I believe this has been fixed in version `FFTrees 1.1.4` so I'll close this for now. If it persists, please open the issue again.

**Comment by bubzebub** [comment]
I can confirm the error has gone away.  There is another issue with only three splits being shown but I will open another issue.
